File: Saood Imran/topic.py
from kafka.admin import KafkaAdminClient, NewTopic

admin = KafkaAdminClient(
    bootstrap_servers=["localhost:9093","localhost:9094","localhost:9095"],
    client_id='test'
)

try:
    topic_list = []
    topic_list.append(NewTopic(name="topicTest", num_partitions=3,replication_factor=1))
    admin.create_topics(new_topics=topic_list, validate_only=False)
except:
    print("An error occured")

File: Saood Imran/topic.js
const {Kafka} = require("kafkajs")
run();
async function run(){ 
    try {
        const kafka = new Kafka({
            "clientID": "myapp",
            "brokers": ["localhost:9093", "localhost:9094", "localhost:9095"]
        })

        const admin = kafka.admin();
        console.log("Connecting..")
        await admin.connect()
        console.log("Connected!")
        await admin.createTopics({
            "topics": [{
                "topic" : "Users",
                "numPartitions": 2
            }]
        })
        console.log("Topics created")
        await admin.disconnect();
    } catch (error) {
        console.error(`Error occured ${ex}`)
    }

    finally{
        process.exit();
    }
}